database/index.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+module.exports = () => {
+  return new Promise((resolve) => {
+    let url = "";
+
+    switch (process.env.ENVIRONMENT) {
+      case "development":
+        url = process.env.DATABASE_URL_DEVELOPMENT;
+        break;
+
+      case "staging":
+        url = process.env.DATABASE_URL_STAGING;
+        break;
+
+      case "test":
+        url = process.env.DATABASE_URL_TEST;
+        break;
+
+      case "production":
+        url = process.env.DATABASE_URL_PRODUCTION;
+        break;
+
+      default:
+        url = process.env.DATABASE_URL_TEST;
```

Comment:
The default connection string should be the development

Reply:
Can we have a local mongodb as well?
So
url = process.env.DATABASE_URL_TEST || 'mongodb://localhost/comcaldb' for running mongo locally

Reply:
@twumm 
 Yes, but you will need to set up a db locally on your machine. view the readme. I will be updating it